config/esw.yaml

```diff
@@ -111,6 +111,40 @@ brushed_motors:
       inversion: 1.0
   i2c_device_file: "/dev/i2c-1"
   test: false
+  
+cameras:
+    arguments:
+      # this has [bps, width, height, fps]
+      worst_res:
+        - "173000"
+        - "320"
+        - "240"
+        - "15"
+      low_res:
+        - "691000"
+        - "640"
+        - "480"
+        - "15"
+      medium_res:
+        - "2000000"
+        - "960"
+        - "720"
+        - "15"
+      high_res:
+        - "3000000"
+        - "1280"
+        - "720"
+        - "15"
+      best_res:
+        - "4200000"
+        - "1280"
+        - "720"
+        - "30"
+    max_video_device_id_number: 10
+    ips:
+      primary: "10.0.0.7"
+      secondary: "10.0.0.8"
+    skip_every_other_device: false
```

Comment:
We should comment what exactly this does and why it's necessary

Reply:
Ok sure. I'll explain now: For some odd reason this year, Ubuntu recognizes each camera device as two devices (/dev/video0 and /dev/video1) but only 0 is the one we want.